### ISSUE TITLE
vscode command to report issue has serious problems

### ISSUE THREAD

**Issue by julie777** [microsoft/vscode#145466]
The above is the problem that I am reporting!

Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.65.2
- OS Version: Windows 10 

Steps to Reproduce:

1. ctrl-shift-p  report issue
2. a local window opens
3. click on some other window
4. report issue windows is gone

### Problem 1
I needed to look at something else in vscode to finish filling out the form. When I clicked outside the form window, the issue window was gone. This happens if I make any other window on the computer active.
I thought that I had lost everything I had typed.
I ran the report issue command again and it opened the form that I was working on so I didn't just give up   
Having the window close without the user causing it directly is very bad!

### Problem 2
After I finished filling out the form I clicked on the submit to github (or whatever it was called)
I got a popup telling me it was to large to send and that I would have to copy it.
I only typed a couple paragraphs. But I guess checking the boxes to automatically include the extra information at the bottom added too much stuff.
So I copied and pasted, and I was shocked to see a bunch of nasty html show up in the window. My words were far down the page.

### Problem 3
Trying to submit an issue on github tells me that I have to use the vscode action. I can't skip all the problems.

## Suggested solution
1. the command just opens a new issue on github or
2. I can open an issue directly on github and
3. you can use a github template for the issue to get the information that you need

**Comment by julie777** [comment]
After thinking about it overnight, I can see two things that would make it useful to report issues from within vscode:
1. vscode can automatically fill in version information, extensions installed, etc. which is useful for tracking down problems
2. users without GitHub accounts can submit issues.

The drawback is that they won't have a chance to search and see if the issue has already been reported. However, given that vscode has over 5000 open issues I don't think anyone is going to do a thorough search anyway.

The problem is trying to submit from vscode and from github at the same time.

### Submit From VScode
I think the solution would be to fill out the issue in vscode and have it automatically be sent/posted on GitHub using special credentials for the purpose, instead of having users even see that part of the process. There is no reason that it should ever be too large to transfer. Vscode is controlling the collecting of the information.

### Submit from GitHub
In addition, for users with GitHub accounts that want to go to GitHub and search issues, etc. and fill submit the issue there you should allow that. Use a template to get the information you need, and add a menu option under help (show system information) to provide the information you need and they can copy and paste it.

(I really do like vscode. I just think it has some warts that need to be removed.)

**Comment by IllusionMH** [comment]
It's better to report different problems in different issues, to simplify tracking different issues by developers.
Bad window behavior is already tracked in #130497

For problem 2 do yoy have example of that "nasty HTML"?
Usually user content is at the top and below is extensions and system info.

Not sure about problem 3,do you mean suggestion in issue template?

**Comment by TylerLeonhardt** [comment]
> users without GitHub accounts can submit issues

This is tracked here: https://github.com/microsoft/vscode/issues/145249

**Comment by TylerLeonhardt** [comment]
> I got a popup telling me it was to large to send and that I would have to copy it.

we open issues like so: https://github.com/microsoft/vscode/issues/new?body=hello

We add extra details that is super useful for us in diagnosing the problem... unfortunately, browsers have a limit to how long a query string can be... which is why we ask for the paste. It's not ideal, but we are waiting on GitHub to give us another option and nothing yet...

I think all the points outlined in this issue are tracked in other issues... so I'm going to close this. If not, can you please open a new issue per-problem you are seeing? It's way easier for us to track... (we can close issues as we fix specific problems rather than having to track a big issue)
